static/src/stylesheets/head.content.scss

```diff
@@ -25,6 +25,7 @@
 @import 'module/content/_recipe-article';
 @import 'module/content/_article-minute';
 @import 'module/content/_gallery.head.scss';
+@import 'module/content/_photo-essay';
```

Comment:
The contents of `head.content.scss` are included on every page. I think there's a case for inlining these styles to keep the CSS weight and parse time down for pages that don't need them